### DISCUSSION TITLE
xformers pypi project capacity is full

### DISCUSSION THREAD

**Discussion by rltgjqmcpgjadyd** [facebookresearch/xformers#965] [General]
please free up capacity to allow wheel uploads

**Comment by danthe3rd** [comment]
I just freed up some wheels.
However the recommend way to install xFormers moving forward is to use the PyTorch pip index, as instructed in the [README](https://github.com/facebookresearch/xformers?tab=readme-ov-file#installing-xformers), eg:
```
# cuda 11.8 version
pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu118
# cuda 12.1 version
pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu121
```

**Reply by rltgjqmcpgjadyd** [reply]
But I want the dev wheel :)
